web/src/features/fireWeather/components/tables/ObservationTable.tsx

```diff
@@ -27,6 +27,12 @@ export const columns: Column[] = [
     align: 'right',
     format: (value: number): number => Math.round(value)
   },
+  {
```

Comment:
Adding this has caused the date to wrap to two lines - I think we're better off not having anything wrap.

Reply:
Fixed!